### DISCUSSION TITLE
Arquero playground?

### DISCUSSION THREAD

**Discussion by mhkeller** [uwdata/arquero#310] [Ideas]
It would be great if there were a place one could try out arquero in the browser similar to the [tidy.js playground](https://pbeshai.github.io/tidy/playground). I often want to check a generic version of a transform function to make sure it works the way I think it does and this would be a great place to do that. The current alternative is using an observable notebook but given that it's not quite javascript and is a separate service, I think something just for arqeuro would be helpful.

**Comment by StephM-W** [comment]
I'm using runkit for this at the moment, for instance here's my current workaround for Arquero's pivot single valueKey simplification _feature_ (mentioned [here](https://github.com/uwdata/arquero/discussions/307)) looks like this: 

https://runkit.com/stephanemw/636aafe5096081000897104e

![image](https://user-images.githubusercontent.com/3695414/206855042-06913915-5621-4743-a109-b9ea65492d85.png)

And the results below
![image](https://user-images.githubusercontent.com/3695414/206855095-ec644a38-ea07-444f-bc04-b6b10f517981.png)
